summerofcode/2022.md

```diff
@@ -117,3 +117,18 @@ If you are a project maintainer and consider mentoring during the GSoC 2022 cycl
 - Mentor(s): Rahul Jadhav (@nyrahul), Barun Acharya (@daemon1024)
 - Upstream Issue (URL): https://github.com/kubearmor/KubeArmor/issues/614
 
+### TiKV
+
+#### Add APM tracing functionality to TiKV Java client
+
+- Description: Add APM tracing functionality to TiKV Java client. Allow send metrics to APM system like jaeger.
+- Recommended Skills: Java, OpenTracing
+- Mentor(s): Xiang Zhang (@zhangyangyu), Xin Yang (@xuanyu66)
+- Upstream Issue (URL): https://github.com/tikv/client-java/issues/513
+
+#### Normalize TiKV Java client for TiSpark
+
+- Description: TiSpark maintains a fork of TiKV Java client of little use and could use the upstream TiKV Java client.
+- Recommended Skills: Java, Spark, TiKV
+- Mentor(s): Xiang Zhang (@zhangyangyu), Yuhang Shi (@shiyuhang0)
+- Upstream Issue (URL): https://github.com/pingcap/tispark/issues/2232
```

Comment:
The upstream issue should be under the project organization (TiKV), not a vendor one.

Reply:
@zhangyangyu let's track the issue to client-java. The work can help avoiding fragmentation of TiKV client java, and it's definitely for the sake of goodness for client java.

Reply:
nailed it